apps/meteor/app/channel-settings/server/functions/saveRoomName.ts

```diff
@@ -73,6 +74,7 @@ export async function saveRoomName(
 
 	if (room.name && !isDiscussion) {
 		await Integrations.updateRoomName(room.name, slugifiedRoomName);
+		void notifyOnIntegrationChangedByChannels([room.name, slugifiedRoomName]);
```

Comment:
in theory we could use only `slugifiedRoomName` right? because after the command `updateRoomName` there should be no references anymore to `room.name`